frontend/public/src/containers/pages/DashboardPage.js

```diff
@@ -234,14 +251,50 @@ export class DashboardPage extends React.Component<
     )
   }
 
+  renderEnrolledItemCard(enrollment: RunEnrollment) {
+    const {
+      programEnrollments,
+      currentUser,
+      deactivateEnrollment,
+      courseEmailsSubscription,
+      addUserNotification,
+    } = this.props
+
+    let showCard = true
+
+    if (programEnrollments) {
+      const enrollmentMatches = programEnrollments.map(programEnrollment => {
+        return programEnrollment.enrollments.some(elem => elem.id === enrollment.id)
+      })
+
+      showCard = !enrollmentMatches.includes(true)
+    }
+
+    if (showCard) {
```

Comment:
Could you invert this if and return null here instead, the less number of lines we have inside the if the better :)

Reply:
Should be done in https://github.com/mitodl/mitxonline/pull/766/commits/358db0bd1fb222d962f565f31d6441be790eb3a1